sql/item_create.cc

```diff
@@ -101,6 +101,19 @@ class Create_sp_func : public Create_qfunc
   Please keep this list sorted in alphabetical order,
   it helps to compare code between versions, and helps with merges conflicts.
 */
+    
```

Comment:
There are 4 whitespaces here, need to be remvoed
Move class definition between `Create_func_floor` and `Create_func_format` as you added in `func_array` and again check for standard recommendations below.